modules/contact-form/grunion-contact-form.php

```diff
@@ -2038,13 +2038,36 @@ function process_submission() {
 		 * with this work around. */
 		add_filter( 'wp_insert_post_data', array( $plugin, 'insert_feedback_filter' ), 10, 2 );
 
+		$disabled_functions = explode( ',', ini_get( 'disable_functions' ) );
+		
+		// Avoid warnings when print_r() has been disabled on hosting provider for security reasons
+		if ( ! in_array( 'print_r', $disabled_functions ) ) {
+			$all_values_string = print_r( $all_values, true );
+		}
+		// If print_r is disabled, try to use var_export as an alternative solution
+		elseif ( ! in_array( 'var_export', $disabled_functions ) ) {
+			$all_values_string = var_export( $all_values, true );
+		}
+		else {
+			$all_values_string = '';
+		}
+		
+		$post_content_string =  $comment_content . "\n<!--more-->\n" . "AUTHOR: {$comment_author}\n
+		AUTHOR EMAIL: {$comment_author_email}\n
```

Comment:
Another thing I had not paid attention to.
To keep the code readable while avoiding inconsistency in formatting, I switched to:

```php
$post_content_string =  $comment_content . "\n<!--more-->\n"
	. "AUTHOR: {$comment_author}\n"
	. "AUTHOR EMAIL: {$comment_author_email}\n"
	. "AUTHOR URL: {$comment_author_url}\n"
	. "SUBJECT: {$subject}\n"
	. "IP: {$comment_author_IP}\n"
	. $all_values_string;
```